packages/apidom-ns-openapi-3-1/src/refractor/specification.ts

```diff
@@ -94,7 +61,7 @@ import WebhooksVisitor from './visitors/open-api-3-1/WebhooksVisitor';
  *
  * Note: Specification object allows to use absolute internal JSON pointers.
  */
-const specification = {
+const specification: any = {
```

Comment:
Added `any` here as otherwise while building the package I was getting: `The inferred type of this node exceeds the maximum length the compiler will serialize. An explicit type annotation is needed`. Can we leave it like this or do we have to explicitly create a type for `specification`?